src/components/Playbooks/Runs/Submit/SubmitPlaybookRunForm.tsx

```diff
@@ -118,12 +110,15 @@ export default function SubmitPlaybookRunForm({
                     <PlaybookSelectResource playbook={playbook} />
                   )
                 )}
+                {/* 
+                  We want hide params until we have a resource selected
+                */}
```

Comment:
I think updating params after resource is selected makes more sense ? 
What do you think @mainawycliffe @moshloop ?

Reply:
I think hiding does make the most sense - we might have different params that are hidden and shown based on the config